activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Support hash as first argument in `assert_difference`. This allows to specify multiple
+    numeric differences in the same assertion.
+    `assert_difference ->{ Article.count } => 1, ->{ Post.count } => 2`
```

Comment:
Can you give this code example its own paragraph and indent it with four spaces, instead of wrapping it in backticks?

http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog

Reply:
Sure!